### ISSUE TITLE
[FEATURE REQUEST] Improvements on `SeeItSayItLabelEnabler` (translation + auto-update)

### ISSUE THREAD

**Issue by anonymous2585** [MixedRealityToolkit/MixedRealityToolkit-Unity#593]
## Describe the problem

1. The "See It Say It Label" doesn't support translations. We'd like the be able to customize the "Say '{keyword}'" pattern depending on the current user language (we use [Unity Localization package](https://docs.unity3d.com/Packages/com.unity.localization@1.5/manual/index.html)).

2. When changing the `SpeechRecognitionKeyword` of a `StatefulInteractable`, the label isn't updated automatically. It could be great if the MRTK handles it natively.

## Describe the solution you'd like

1. A translation compatibility with the Unity Localization package: if the package is installed, display a LocalizedString field in the inspector to allow to choose the pattern (using a smart string) for the keyword.

2. Add SpeechRecognitionKeyword changed events in the `StatefulInteractable` and register the `SeeItSayItLabelEnabler` to it so it automatically update

**Comment by AMollis** [comment]
Good idea.  We welcome community contributions to the project.  If you have a solution for this, please purpose to the project maintainers.

**Comment by AMollis** [comment]
Hi @anonymous2585 ,

Thank you for reporting this issue or requesting this feature. We appreciate your feedback and interest in MRTK3.

We have reviewed your issue or feature request and confirmed that it is valid. We have added it to our backlog and will prioritize it accordingly.

However, if you would like to speed up the process and contribute to the MRTK3 project, you are welcome to submit a Pull Request with your proposed fix or implementation. We would be happy to review your code and merge it into the main branch if it meets our quality standards.

Please let us know if you have any questions or need any assistance. Thank you for your support and collaboration.

Best regards,
The MRTK3 team
